src/server/config.rs

```diff
@@ -365,6 +391,10 @@ impl Config {
             ));
         }
 
+        if self.end_point_memory_quota.is_none() {
+            self.end_point_memory_quota = Some(*DEFAULT_ENDPOINT_MEMORY_QUOTA);
```

Comment:
should we do some basic checking? for example it should be less than `SysQuota::memory_limit_in_bytes()- block cache capacity`?